process/smartContract/hooks/blockChainHook.go

```diff
@@ -625,9 +627,7 @@ func (bh *BlockChainHookImpl) GetCompiledCode(codeHash []byte) (bool, []byte) {
 func (bh *BlockChainHookImpl) DeleteCompiledCode(codeHash []byte) {
 	bh.compiledScPool.Remove(codeHash)
 	err := bh.compiledScStorage.Remove(codeHash)
-	if err != nil {
-		log.Debug("DeleteCompiledCode", "error", err, "codeHash", codeHash)
-	}
+	log.LogIfError(err, "codeHash", codeHash)
```

Comment:
`log.LogIfError(err, "func", "BlockChainHookImpl.DeleteCompiledCode: compiledScStorage.Remove")`

Reply:
Right. Missed it. Added it